lib/support/incrementVersion.js

```diff
@@ -1,4 +1,8 @@
 var version = 1;
 module.exports = function incrementVersion() {
+    // console.log(new Error().stack);
```

Comment:
Minor: lingering comment

Reply:
thanks for spotting!

Reply:
Otherwise LGTM. Feel free to self-merge.